### PR TITLE
Rsa 2216 imply

### PR DESCRIPTION
**Pull request by steventranjs**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)
- [x] Add docker based image for imply

## How was this tested?
- [x] Manually: http://ec2-54-213-204-124.us-west-2.compute.amazonaws.com:8090/console.html

## Related Links
https://jira.ruckuswireless.com/browse/RSA-2216

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,20 @@
 FROM ubuntu:16.04
 
+ENV IMPLY_VERSION 2.2.3
```
Comment (almike) [inline Dockerfile]: actually we can use ARG IMPLY_VERSION here, then pass it to the build command:
https://docs.docker.com/engine/reference/builder/#arg

that way when we want to update we just need to update the circleci file.

File: Dockerfile [hunk 2]
```diff
@@ -1,16 +1,20 @@
 FROM ubuntu:16.04
 
+ARG IMPLY_VERSION
+RUN apt-get update && apt-get install -y cron wget
```
Comment (almike) [inline Dockerfile]: && apt-get clean

Comment (almike) [reply Dockerfile]: it does not help at all to add clean

**Review by almike — APPROVED** [review]
tried it with `sudo docker run --env-file .env -p 8081-8110:8081-8110 -p 2181:2181 -d --name imply-master imply-base`

.env file

```
IMPLY_DB_NAME=druid
IMPLY_DB_PASS=diurd
IMPLY_DB_USER=druid
IMPLY_ZOOKEEPER_HOST=10.0.2.15
IMPLY_ZOOKEEPER_PORT=2181
POSTGRESQL_HOST=10.0.2.15
POSTGRESQL_PORT=5432
HDFS_HOST=10.0.2.15
```
 the coordinator and overlord UIs were up

**Review by jasoncheow — APPROVED** [review]
LGTM

**Pull request merged**